src/module/helperCode/Equipment Checker.py

```
@@ -40,54 +40,54 @@ def magicPrice(price, magicBonus):
                 
-                if data["type"] == "equipment" and data['folder'] != "K5xsW4FLQleQsk3D":
-                    if data["system"]["type"]["value"] in ["shield", "heavy", "medium", "light"]:
-                        # Applies the default magic bonus if in the name
-                        data["system"]["armor"]["magicalBonus"] = magicBonus
-                        data["system"]["unidentified"]["description"] = "<p><em>This armor is unusually strong and well-crafted. It is most likely magic, though you can’t identify the specific enchantment.</em></p>"
-                        data["system"]["source"]["custom"] = "elkan5e.com/weapons"
+                # if data["type"] == "equipment" and data['folder'] != "K5xsW4FLQleQsk3D":
+                #     if data["system"]["type"]["value"] in ["shield", "heavy", "medium", "light"]:
+                #         # Applies the default magic bonus if in the name
+                #         data["system"]["armor"]["magicalBonus"] = magicBonus
+                #         data["system"]["unidentified"]["description"] = "<p><em>This armor is unusually strong and well-crafted. It is most likely magic, though you can’t identify the specific enchantment.</em></p>"
```

Ruff (0.8.2)

Comment contains ambiguous `’` (RIGHT SINGLE QUOTATION MARK). Did you mean ``` (GRAVE ACCENT)?

(RUF003)